scss/jquery/overrides/_ui-datepicker.scss

```diff
@@ -0,0 +1,174 @@
+$datepicker-border-color: rgba(0, 0, 0, 0.15);
+$datepicker-gray-dark: #535a67;
+$datepicker-gray-darker: #525966;
+
+.ui-datepicker {
+  background: $crm-white !important;
+  border: 1px solid $datepicker-border-color !important;
+  border-radius: 0;
+  box-shadow: $box-shadow !important;
+  min-width: 160px;
+  padding: 10px !important;
+  width: auto !important;
+  z-index: 9999 !important;
+
+  &,
+  * {
+    @include open-sans-font-family();
+
+    border-radius: 0;
+  }
+
+  *:not(b) {
+    background: none !important;
+  }
+
+  .ui-datepicker-header {
+    border: 0;
+    margin-bottom: 5px;
+
+    .ui-datepicker-prev,
+    .ui-datepicker-next {
+      border-radius: 2px;
+      box-shadow: none !important;
+      cursor: pointer;
+      text-align: center;
+      top: 4px;
+      width: 35px;
+
+      &,
+      &.ui-state-hover {
+        height: 28px;
+      }
+
+      &::before {
+        color: $datepicker-gray-dark !important;
+        display: inline-block;
+        font-size: 14px;
+        font-weight: normal;
+        line-height: 30px;
+        padding: 0 !important;
+      }
+
+      &-hover {
+        background-color: #e6e6e6 !important;
+        border: 0 !important;
+        border-color: $datepicker-gray-darker;
+        color: $datepicker-gray-dark;
+      }
+    }
+
+    select {
+      @extend .form-control; /* stylelint-disable-line scss/at-extend-no-missing-placeholder */
+      @include prefix(appearance, none, webkit moz);
+
+      background: url('../svgs/drop-down-arrow.svg') no-repeat right !important;
+      display: inline-block;
+      font-size: 0.8em;
+      padding-left: 5px !important;
+      padding-right: 10px !important;
+      position: relative;
+      top: 1px;
+      width: 75px;
+    }
+  }
+
+  .ui-datepicker-prev {
+    &::before {
+      @include fa-icon(13px, '\f053');
+    }
+
+    &-hover {
+      left: 2px !important;
+    }
+  }
+
+  .ui-datepicker-next {
+    &::before {
+      @include fa-icon(13px, '\f054');
+    }
+
+    &-hover {
+      right: 2px !important;
+    }
+  }
+
+  .ui-datepicker-title {
+    padding: 0 30px;
+
+    .select2-container {
+      background: $crm-white;
+      margin: 0 5px;
+    }
+
+    .select2-choice {
+      background: none;
+      border: 1px solid $crm-grayblue-darker;
+      border-radius: 2px;
+      box-shadow: $crm-select2-box-shadow-inset;
+      font-weight: 400 !important;
+    }
+
+    .select2-arrow {
+      background: none;
+      padding: 0 5px;
+    }
+  }
+
+  .ui-state-default {
+    border: 0 !important;
+  }
+
+  .ui-state-disabled .ui-state-default {
+    color: $crm-gray-matte;
+    padding: 7px;
+  }
+}
+
+.ui-datepicker-calendar {
+  border: 0 !important;
+
+  thead {
+    tr {
+      border-top: 1px solid rgb(221, 221, 221);
+    }
+
+    th {
+      color: $datepicker-gray-darker;
+      font-size: 11px;
+    }
+  }
+
+  [data-handler],
```

Comment:
this looks weird, what's this for?

Reply:
This for the dates which are not disabled, and they dont have a class
![2018-03-19 at 4 39 pm](https://user-images.githubusercontent.com/5058867/37592384-7d5add78-2b94-11e8-94f2-6ba319f0fc7a.png)

Reply:
leave a comment explaining that then, please